daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/ScriptF.scala

```diff
@@ -1051,6 +1116,38 @@ object ScriptF {
       case _ => Left(s"Expected ListUserRights payload but got $v")
     }
 
+  private def parseChangePackages(
+      ctx: Ctx,
+      v: SValue,
+  ): Either[String, (List[ScriptLedgerClient.ReadablePackageId], StackTrace, SValue)] = {
+    def toReadablePackageId(s: SValue): Either[String, ScriptLedgerClient.ReadablePackageId] =
+      s match {
+        case SRecord(_, _, ArrayList(SText(name), SText(version))) =>
+          for {
+            pname <- PackageName.fromString(name)
+            pversion <- PackageVersion.fromString(version)
+          } yield ScriptLedgerClient.ReadablePackageId(pname, pversion)
+        case _ => Left(s"Expected PackageName but got $s")
+      }
+    v match {
+      case SRecord(_, _, ArrayList(packages, continue, stackTrace)) =>
+        for {
+          packages <- Converter.toList(packages, toReadablePackageId)
+          stackTrace <- toStackTrace(ctx, Some(stackTrace))
+        } yield (packages, stackTrace, continue)
+      case _ => Left(s"Expected Packages payload but got $v")
+    }
+  }
+
+  private def parseListPackages(ctx: Ctx, v: SValue): Either[String, ListPackages] =
+    v match {
+      case SRecord(_, _, ArrayList(continue, stackTrace)) =>
+        for {
+          stackTrace <- toStackTrace(ctx, Some(stackTrace))
```

Comment:
Do we even need stackTrace? Other Cmd's have this and don't appear to use it, unsure if theres intent to use it in future, or if its legacy